### DISCUSSION TITLE
ストックカテゴリーの削除、リネーム操作をわかりやすくしてほしい

### DISCUSSION THREAD

**Discussion by ghost** [increments/qiita-discussions#227] [要望・アイデア・不具合]
[ストック一覧ページ](https://qiita.com/stock)において、ストックの削除とリネームをどこから行えばいいのか、などがわかりにくいと感じました。

## 理由
- アイコンが小さく、目立たない
- カテゴリを新規作成するときと操作する場所が違う
-  カテゴリの`すべて` ,`あとで読む`を表示している場合にはアイコンが表示されない
- テキストやツールチップがない
- 操作の対象が明確でない
  - 特にゴミ箱アイコンがカテゴリを削除するのか、記事をストックから削除するのかわかりにくい
 
![image](https://user-images.githubusercontent.com/71774299/169640694-343d892e-7642-49a8-944b-516e39f5b561.png)

## 改善案
あくまで一例ですが、
- アイコンをテキストまたはツールチップをつけた大きめのボタンにする
- カテゴリ新規作成のボタンを用意して、リネーム、削除アイコンの近くに配置する
- 削除、リネームアイコンを押した後に対象のカテゴリを選択するようにする
  - これなら`すべて` ,`あとで読む`の場所にもアイコンを表示しても問題ないはず

**Comment by tanaka-asami415** [comment]
@sin471 

ご意見ありがとうございます。
ご不便をおかけして申し訳ありません🙏

ストック関連はご要望も多く、我々としてもしっかりと改善に取り組みたいと思っていますので
今回いただいたご意見も一緒に検討させていただきます！🙇

#229
みなさんの意見を集約して、仕様を決めていきますので
もうしばらくお待ちいただけると幸いです🙏

**Comment by mkdk09** [comment]
大変お待たせしました！🙇

カテゴリーの名前変更と削除の操作をわかりやすくするために、アイコンの横にラベルを追加しました。
また、タップしやすいようにアイコンを右端に移動させました。
